.github/workflows/release.yml

```diff
@@ -10,10 +10,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rgd-module: ['django-rgd', 'django-rgd-3d', 'django-rgd-fmv', 'django-rgd-geometry', 'django-rgd-imagery', 'rgd-client']
+        rgd-module: ['django-rgd', 'django-rgd-client', 'django-rgd-3d', 'django-rgd-fmv', 'django-rgd-geometry', 'django-rgd-imagery']
         include:
           - rgd-module: django-rgd
             TWINE_PASSWORD_VAR: PYPI_API_TOKEN_RGD
+          - rgd-module: django-rgd-client
+            TWINE_PASSWORD_VAR: PYPI_API_TOKEN_RGD_CLIENT
```

Comment:
I think here, you want to do `django-rgd/client` instead of `django-rgd-client` to set the right working directory.

Reply:
I think we also need to publish the `rgd-imagery-client` package under `django-rgd-imagery/client`

Reply:
I've made these changes, but I'm assuming you'll want to create the `rgd-imagery-client` package yourself, since I believe you have control of the other PyPI packages.

Reply:
Great, thanks - I will handle publishing to PyPI after this is merged and we're ready